cmd/mysterium_server/command_run/command.go

```diff
@@ -92,14 +93,14 @@ func (cmd *CommandRun) Run() (err error) {
 	return nil
 }
 
-func detectCountry() (string, error) {
+func detectCountry(detector location.Detector) (string, error) {
 	ipifyClient := ipify.NewClient()
```

Comment:
`ipifyClient` is already injected into command

Reply:
Right! Updated to re-use `ipifyClient`